mito-ai/src/Extensions/AiChat/ChatMessage/ChatInput.tsx

```diff
@@ -62,13 +186,21 @@ const ChatInput: React.FC<ChatInputProps> = ({
                     }
                 }}
             />
-            {isEditing && 
+            {isEditing &&
                 <div className="message-edit-buttons">
                     <button onClick={() => onSave(input)}>Save</button>
                     <button onClick={onCancel}>Cancel</button>
                 </div>
             }
-        </>
+            {isDropdownVisible && !showDropdownAbove && (
+                <ChatDropdown
+                    options={expandedVariables}
+                    onSelect={handleOptionSelect}
+                    filterText={dropdownFilter}
+                    position="below"
+                />
+            )}
```

Comment:
I'm still pretty sure we only need one of these. I'm going to try removing it and seeing what happens.